util/MsSql/backup-db.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/sh
 
-# Delete backup files older than 30 days
-find /etc/bitwarden/mssql/backups/ -type f -name '*.BAK' -mindepth 1 -mtime +30 -delete
+# Backup timestamp
+export now=${1:-$(date +%Y%m%d_%H%M%S)}
```

Comment:
Not sure why a user would every want to pass in a value. I would think it should always default to now date